src/components/VendorSalesPage/VendorSalesPage.js

```diff
@@ -83,17 +84,230 @@ class VendorSalesPage extends Component {
+		<Option data-countryCode="SH" value="290">St. Helena (+290)</Option>
+		<Option data-countryCode="KN" value="1869">St. Kitts (+1869)</Option>
+		<Option data-countryCode="SC" value="1758">St. Lucia (+1758)</Option>
+		<Option data-countryCode="SD" value="249">Sudan (+249)</Option>
+		<Option data-countryCode="SR" value="597">Suriname (+597)</Option>
+		<Option data-countryCode="SZ" value="268">Swaziland (+268)</Option>
+		<Option data-countryCode="SE" value="46">Sweden (+46)</Option>
+		<Option data-countryCode="CH" value="41">Switzerland (+41)</Option>
+		<Option data-countryCode="SI" value="963">Syria (+963)</Option>
+		<Option data-countryCode="TW" value="886">Taiwan (+886)</Option>
+		<Option data-countryCode="TJ" value="7">Tajikstan (+7)</Option>
+		<Option data-countryCode="TH" value="66">Thailand (+66)</Option>
+		<Option data-countryCode="TG" value="228">Togo (+228)</Option>
+		<Option data-countryCode="TO" value="676">Tonga (+676)</Option>
+		<Option data-countryCode="TT" value="1868">Trinidad &amp; Tobago (+1868)</Option>
+		<Option data-countryCode="TN" value="216">Tunisia (+216)</Option>
+		<Option data-countryCode="TR" value="90">Turkey (+90)</Option>
+		<Option data-countryCode="TM" value="7">Turkmenistan (+7)</Option>
+		<Option data-countryCode="TM" value="993">Turkmenistan (+993)</Option>
+		<Option data-countryCode="TC" value="1649">Turks &amp; Caicos Islands (+1649)</Option>
+		<Option data-countryCode="TV" value="688">Tuvalu (+688)</Option>
+		<Option data-countryCode="UG" value="256">Uganda (+256)</Option>
+	  <Option data-countryCode="GB" value="44">UK (+44)</Option>
+		<Option data-countryCode="UA" value="380">Ukraine (+380)</Option>
+		<Option data-countryCode="AE" value="971">United Arab Emirates (+971)</Option>
+		<Option data-countryCode="UY" value="598">Uruguay (+598)</Option>
+		<Option data-countryCode="US" value="1">USA (+1)</Option> 
+		<Option data-countryCode="UZ" value="7">Uzbekistan (+7)</Option>
+		<Option data-countryCode="VU" value="678">Vanuatu (+678)</Option>
+		<Option data-countryCode="VA" value="379">Vatican City (+379)</Option>
+		<Option data-countryCode="VE" value="58">Venezuela (+58)</Option>
+		<Option data-countryCode="VN" value="84">Vietnam (+84)</Option>
+		<Option data-countryCode="VG" value="84">Virgin Islands - British (+1284)</Option>
+		<Option data-countryCode="VI" value="84">Virgin Islands - US (+1340)</Option>
+		<Option data-countryCode="WF" value="681">Wallis &amp; Futuna (+681)</Option>
+		<Option data-countryCode="YE" value="969">Yemen (North)(+969)</Option>
+		<Option data-countryCode="YE" value="967">Yemen (South)(+967)</Option>
+		<Option data-countryCode="ZM" value="260">Zambia (+260)</Option>
+		<Option data-countryCode="ZW" value="263">Zimbabwe (+263)</Option>
+    </Select>
       );
 
       return (
         <Fragment>
-               {msg ? <Alert message={msg} type="error" /> : null}
+							 {msg ? <Alert message={msg} type={alertType} /> : null}
```

Comment:
@ritwizsinha What is this use of alertType ? You are only keeping it empty or error.

Reply:
done